src/app/db/tables.py

```diff
@@ -27,7 +27,7 @@
     "accesses",
     metadata,
     Column("id", Integer, primary_key=True),
-    Column("login", String(50), unique=True),
+    Column("login", String(50), unique=True, index=True),  # index for fast lookup
```

Comment:
does this mean that it won't be fast to access using the primary key?

Reply:
It won't slow down the access via primary key, it will maintain a secondary index (it means a little bit more storage) but allows "random" access (opposite of sequential). I am investigating about difference between unique contraint and unique index for postgres and sqlalchemy, terms are quite confusing

Reply:
Thanks!